### PR TITLE
dev: exclude the blackening from blames via .git-blame-ignore-revs

### PR DESCRIPTION
**Pull request by joshuarli**
This `.git-blame-ignore-revs` file path relative to the git root seems to be a common pattern across various software projects. Using it to ignore the large blackening commit so people don't mistakenly ask me about code I didn't actually write, haha.

See https://reviews.llvm.org/D67145.

### CODE REVIEW

**Comment by billyvg** [comment]
👍 we should add some of the prettier ones too, I'll dig some up in a bit

**Comment by joshuarli** [comment]
@billyvg Feel free to tack those on to this branch!

File: Makefile [hunk 1]
```diff
@@ -52,10 +52,13 @@ ensure-venv:
 ensure-pinned-pip: ensure-venv
 	$(PIP) install --no-cache-dir --upgrade "pip>=20.0.2"
 
-setup-git: ensure-venv
+setup-git-config:
+	@git config branch.autosetuprebase always
+	@git config core.ignorecase false
+	@git config --local blame.ignoreRevsFile .git-blame-ignore-revs
```
Comment (EvanPurkhiser) [inline Makefile:58]: I lied about this `--local` thing, I don't think that exists

**Comment by EvanPurkhiser** [comment]
Let's slam this in as is and billy can add later

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**